tests/algorithms/test_torch_export.py

```diff
@@ -120,18 +120,20 @@ def test_surgery_torchfx_eval(name, surgery_method, input):
     pytest.param("channels_last", apply_channels_last)
 ])
 @pytest.mark.timeout(10)
-def test_surgery_onnx(name, surgery_method, input, tmpdir):
+@pytest.mark.filterwarnings(
+    r"ignore:Converting a tensor to a Python .* might cause the trace to be incorrect:torch.jit._trace.TracerWarning")
+def test_surgery_onnx(name, surgery_method, input, tmp_path):
     """Tests onnx export and runtime"""
     pytest.importorskip("onnx")
-    import onnx
-    import onnxruntime as ort
+    import onnx  # type: ignore
```

Comment:
ditto comment from before

Reply:
Same as https://github.com/mosaicml/composer/pull/1001/files#r866082405

Reply:
Removed the `# type: ignore`; switched the pyproject.toml to ignoring the error